index.bs

```diff
@@ -1625,6 +1639,14 @@ partial interface MLGraphBuilder {
 };
 </script>
 
+<dl dfn-type=dict-member dfn-for=MLClampOptions>
+    : <dfn>minValue</dfn>
+    :: The minimum value of the range. When it is not specified, the clamping is not performed on the lower limit of the range.
+
+    : <dfn>maxValue</dfn>
+    :: The maximum value of the range. When it is not specified, the clamping is not performed on the upper limit of the range.
```

Comment:
I had to stop and think if "the clamping is not performed" was really equivalent to what happens in actual practice that "the unspecified limit is treated as infinity for the data type" (or rather negative infinity for the minimum), and I realize it is, because in both cases if the input is less than infinity, then it's equivalent, but also if NaN, then the output will be NaN, and if the input is infinity, the output will be infinity. So, it's equivalent.